common/validate.go

```diff
@@ -99,9 +99,9 @@ func ValidatePush(spec v1alpha1.OCIBuilderSpec) error {
 }
 
 // ValidatePushSpec validates the lower level push specification
-func ValidatePushSpec(spec v1alpha1.PushSpec) error {
+func ValidatePushSpec(spec *v1alpha1.PushSpec) error {
 	if spec.Registry == "" {
-		return errors.New("push registry must be specified for push")
+		spec.Registry = "docker.io"
```

Comment:
can you make "docker.io" a constant as `DefaultImageRegistry` or something like that?

Reply:
done